task2.Rmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "task2"
+author: "Jan Prugarewicz"
+date: "25 11 2020"
+output: html_document
+---
+
+# Zadanie 2
```

Comment:
wszystko to musi znaleźć się w tym samym pliku raportowym co zadanie 1, żeby powstał jeden spójny raport